static/app/components/modals/widgetViewerModal/widgetViewerTableCell.tsx

```diff
@@ -37,7 +39,32 @@ type Props = {
   tableData?: TableDataWithTitle;
 };
 
-export const renderGridHeaderCell =
+export const renderIssueGridHeaderCell =
+  ({location, widget, tableData}: Props) =>
+  (column: TableColumn<keyof TableDataRow>, _columnIndex: number): React.ReactNode => {
+    const tableMeta = tableData?.meta;
+    const align = fieldAlignment(column.name, column.type, tableMeta);
+    const sortField = getSortField(String(column.key));
+
+    return (
+      <SortLink
+        align={align}
+        title={<StyledTooltip title={column.name}>{column.name}</StyledTooltip>}
+        direction={widget.queries[0].orderby === sortField ? 'desc' : undefined}
+        canSort
+        generateSortLink={() => {
+          return sortField
+            ? {
+                ...location,
+                query: {...location.query, [WidgetViewerQueryField.SORT]: sortField},
```

Comment:
Should we be negating the sort here if we are sorting by this field? Currently I'm unable to reverse the sorting.

Reply:
Hey Nar, Issues API doesn't support sorting in ascending order at this moment 😞 .
The api only supports a limited set of options, namely descending Events, descending Users, First Seen, and Last Seen (none of these in reverse order).
Due to this, we are only able to allow limited sorting for Issue Widgets in the Viewer.